packages/pwa-kit-react-sdk/src/ssr/universal/components/route-component/index.js

```diff
@@ -54,6 +54,7 @@ const withErrorHandling = (Wrapped) => {
  * that can be used to fetch data on the server and on the client, seamlessly.
  */
 export const routeComponent = (Wrapped, isPage, locals) => {
+    const AppConfig = getAppConfig()
```

Comment:
So the one thing I like about this PR is directly related to the thing I don't necessarily like about it. 

This comment isn't about this line specifically, but with regards to this file. 

One of the problems I got myself into originally was getting too deep into splitting up with component into is difference shared responsibilities (e.g. error handling, loadable getComponent stuff etc) in order to pull out the 'getProps' stuff. It looks like you've either chosen to leave that work out of this PR to do later, or decided to ignore it. 

So long storey short, I like that this solution is relatively simple looking. But I don't like that technically `getProps` is something that still lives on the client side, it's only optional for server-side rendering. (Feel free to correct me if I'm wrong, but it looks like that is the case).